tests/gis_tests/inspectapp/tests.py

```diff
@@ -141,8 +141,10 @@ def test_time_field(self):
         else:
             self.assertIn('    f_decimal = models.DecimalField(max_digits=0, decimal_places=0)', model_def)
         self.assertIn('    f_int = models.IntegerField()', model_def)
-        self.assertIn('    f_datetime = models.DateTimeField()', model_def)
-        self.assertIn('    f_time = models.TimeField()', model_def)
+        if connection.vendor != 'mysql' or connection.mysql_is_mariadb:
```

Comment:
I'm not sure if this skipping logic is correct -- it seems to skip on MySQL but not MariaDB.